spring-xd-codec/src/main/java/org/springframework/xd/dirt/integration/bus/serializer/kryo/AbstractKryoCodec.java

```diff
@@ -38,17 +38,6 @@
  */
 public abstract class AbstractKryoCodec<T> extends AbstractCodec<T> {
```

Comment:
Related to the optimization work for serialization/deserialization that would go into another PR, AbstractCode always using a ByteArrayInputStream might not be a good idea.  Perhaps we should change this abstract base class to an interface.